Assets/Scripts/Input/Player.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/// <summary>
+/// Script controls player character behaviour
+/// </summary>
+public class Player : MonoBehaviour
+{
+    #region Fields
+
+    public float runningVelocity;
+    public float fallVelocityLimit;
+    public float jumpVelocity;
+
+    public float yAccel;
+    private float yVelo = 0f;
+    private bool grounded = false;
+    private bool jumpingGrace = false;
```

Comment:
What exactly does jumpingGrace mean / do?

Reply:
Jumping grace is a period of time in which the detection of floors to land on is prevented. It's used to allow the player to jump and shortly after still be able to land.
Also see jumpingGraceTime.